src/SFA.DAS.EmployerAccounts.Web/Controllers/SearchPensionRegulatorController.cs

```diff
@@ -99,9 +103,19 @@ public async Task<ActionResult> SearchPensionRegulator(string hashedAccountId, S
         }
 
         [HttpGet]
-        [Route("pensionregulator/aorn")]
-        public async Task<ActionResult> SearchPensionRegulatorByAorn(string payeRef, string aorn)
+        [Route("{HashedAccountId}/pensionregulator/aorn", Order = 0)]
+        [Route("pensionregulator/aorn", Order = 1)]
+        public async Task<ActionResult> SearchPensionRegulatorByAorn(string payeRef, string aorn, string hashedAccountId)
         {
+            if (!string.IsNullOrWhiteSpace(hashedAccountId))
```

Comment:
So is the cookie a removal of the same cookie which is then re-added, or are we removing a cookie which we want to overwrite with the hashed account id? ( Appreciate this is mimicking the GovernmentGateway signin page, just trying to understand it)